### ISSUE TITLE
There should be no empty links

### ISSUE THREAD

**Issue by mgifford** [elmsln/elmsln#2466]
Bug:
- ELMSLN version/commit reported on admin/reports/status

Latest commit 0be9880 on Jan 26

- What happens

This is reported as an empty link with WebAim's WAVE toolbar.  The title applied to the `<paper-button>` shows up fine on hover, but a screen reader won't know that this is an edit link. 

```
<a tabindex="-1" id="lrnsys-button-link" class="style-scope lrnsys-button" target="" href="/user/1/edit"><paper-button id="button" class="style-scope lrnsys-button x-scope paper-button-0" role="button" tabindex="0" animated="" aria-disabled="false" elevation="0" title="Edit content"><div class="inner no-padding style-scope lrnsys-button"><iron-icon id="icon" class="style-scope lrnsys-button x-scope iron-icon-1"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z" class="style-scope iron-icon"></path></g></svg></iron-icon><span class="label style-scope lrnsys-button" hidden=""></span>
              </div><paper-ripple class="style-scope paper-button"><div id="background" class="style-scope paper-ripple"></div><div id="waves" class="style-scope paper-ripple"></div></paper-ripple></paper-button></a>
```

- What you expected to happen

I think the best approach here is actually to apply a title to the SVG & do this consistently for all of your SVG files. There are some good best practices for [SVG files that I've applied some changes](https://www.sitepoint.com/tips-accessible-svg/):

```
<svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;" role="img">
<title id="edit-icon-title">Edit content</title>
<g class="style-scope iron-icon"><path d="M3 17.25V21h3.75L17.81 9.94l-3.75-3.75L3 17.25zM20.71 7.04c.39-.39.39-1.02 0-1.41l-2.34-2.34c-.39-.39-1.02-.39-1.41 0l-1.83 1.83 3.75 3.75 1.83-1.83z" class="style-scope iron-icon"></path></g>
</svg>
```

Feature request:
- What you are trying to do / where it should go

Make sure that there is text inside the link that is available to screen readers. Titles in SVG images are good.

**Comment by mgifford** [comment]
Same issue with **Less common operations**

```
<a tabindex="-1" id="lrnsys-button-link" class="style-scope lrnsys-button" target="" href="#"><paper-button id="button" class="style-scope lrnsys-button x-scope paper-button-0" role="button" tabindex="0" animated="" aria-disabled="false" elevation="0" title="Less common operations"><div class="inner style-scope lrnsys-button"><iron-icon id="icon" class="style-scope lrnsys-button x-scope iron-icon-1"><svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope iron-icon" style="pointer-events: none; display: block; width: 100%; height: 100%;"><g class="style-scope iron-icon"><path d="M12 8c1.1 0 2-.9 2-2s-.9-2-2-2-2 .9-2 2 .9 2 2 2zm0 2c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2zm0 6c-1.1 0-2 .9-2 2s.9 2 2 2 2-.9 2-2-.9-2-2-2z" class="style-scope iron-icon"></path></g></svg></iron-icon><span class="label style-scope lrnsys-button" hidden=""></span>
              </div><paper-ripple class="style-scope paper-button"><div id="background" class="style-scope paper-ripple"></div><div id="waves" class="style-scope paper-ripple"></div></paper-ripple></paper-button></a>
```

**Comment by btopro** [comment]
I'll have to see how their templates are applied and the interplay between the svg and the title. web components abstract a bit of that flexibility away but I'm sure it's possible. Thanks for the feedback

**Comment by btopro** [comment]
V1 elements should resolve this
